src/components/Router/StepComponentMap.js

```diff
@@ -39,9 +40,10 @@ const createDocumentComponents = (documentType) => {
   return frontDocumentFlow
 }
 
-const crossDeviceSteps = (hasComplete) => {
+const crossDeviceSteps = (steps) => {
   const baseSteps = [{'type': 'crossDevice'}]
-  return hasComplete ? [...baseSteps, {'type': 'complete'}] : baseSteps
+  const completeStep = steps.find((step) => step.type === 'complete')
+  return hasCompleteStep(steps) ? [...baseSteps, completeStep] : [...baseSteps]
```

Comment:
why `[...baseSteps]`? Not really a big deal, but it's inconsistent with the other methods - and looks unnecessary? (i.e. just `baseSteps` instead would be sufficient, no?)